src/experts/UnDeserializableValue.js

```diff
@@ -0,0 +1,80 @@
+( function( $, vv ) {
+	'use strict';
+
+	var PARENT = vv.Expert;
+
+	/**
+	 * `Valueview` expert for displaying (or rather not displaying) a data value not supported by
+	 * the `valueview` UI because there is not specialised expert devoted to that data value type.
+	 * @class jQuery.valueview.experts.UnsupportedValue
+	 * @extends jQuery.valueview.Expert
+	 * @since 0.1
+	 * @licence GNU GPL v2+
+	 * @author Daniel Werner < daniel.werner@wikimedia.de >
+	 * @author Katie Filbert < aude.wiki@gmail.com >
+	 */
+	vv.experts.UnDeserializableValue = vv.expert( 'UnDeserializableValue', PARENT, {
+		/**
+		 * @inheritdoc
+		 * @protected
+		 */
+		_options: {
+			messages: {
+				'valueview-expert-undeserializablevalue':
+					'The value is invalid and cannot be displayed.'
```

Comment:
i18n messages get merged in and applied if mediawiki is available. i think this is just a default or something.  

anyway, i have changed the code to reuse the existing formatted value (error message) which means we don't need the same message twice (once in wikibase, and once here which then never gets translated because this component is not in gerrit)